### PR TITLE
popsborder install instructions for conda on Windows

### PR DESCRIPTION
**Pull request by arielsaffer**

### CODE REVIEW

File: docs/conda_windows_install.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+As an alternative to pipenv, you can manage the Python packages needed to run popsborder using a conda environment.
+
+Unless you plan to use additional features of the Anaconda suite (e.g. Anaconda Navigator, Jupyter Notebook, and other tools), we recommend installing Miniconda, which contains only the conda package manager and Python.
+
+Follow the instruction to download and install Miniconda: https://docs.conda.io/en/latest/miniconda.html
+
+Once installed, open Command Prompt and run the following lines, one at a time:
+
+```
```
Comment (wenzeslaus) [inline docs/conda_windows_install.md:11]: The Super-Linter wants to know the language of the code block here.  I'm not sure what is appropriate for Windows command line. Maybe `bat` is okay here (we render it on GitHub, so their [Batchfile](https://github.com/github/linguist/blob/73e2d735c3c26577fc89c1cb3f8342e5a8ff1d82/lib/linguist/languages.yml#L484) category should apply).

The syntax is:

````
```bat
conda...
```
````

```
 /github/workspace/docs/conda_windows_install.md:11 MD040/fenced-code-language Fenced code blocks should have a language specified [Context: "```"]]
```

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
Thank you. This looks good. I'm not sure how to resolve the tension between miniconda and the need for notebooks, but I would leave it as is here. I'm mainly concerned with the list of packages now.

File: docs/conda_windows_install.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+As an alternative to pipenv, you can manage the Python packages needed to run popsborder using a conda environment.
+
+Unless you plan to use additional features of the Anaconda suite (e.g. Anaconda Navigator, Jupyter Notebook, and other tools), we recommend installing Miniconda, which contains only the conda package manager and Python. 
+
+Follow the instruction to download and install Miniconda: https://docs.conda.io/en/latest/miniconda.html
+
+Once installed, open Command Prompt and run the following lines, one at a time: 
+
+```
+conda create --name popsborder_env python=3.8 pytest pylint flake8 black matplotlib seaborn pyyaml git pip
```
[the remark this answers is not in the document]
Comment (arielsaffer) [reply docs/conda_windows_install.md:12]: That makes sense! I updated the package list/Python version accordingly. Re-the above comment, I added a line to install jupyterlab, which I think should be enough to use notebooks from Anacaonda Prompt.

**Review by wenzeslaus — APPROVED** [review]
Great. Thank you!

**Pull request merged**